src/lib.rs

```diff
@@ -20,27 +20,14 @@ use std::{
 
 pub use wgt::*;
 pub use wgc::{
-    Extent3d,
-    Origin3d,
-    command::{
-        CommandBufferDescriptor,
-    },
+    binding_model::TextureComponentType,
     device::{
         BIND_BUFFER_ALIGNMENT,
     },
     instance::{
         AdapterInfo,
         DeviceType,
     },
-    resource::{
```

Comment:
how come we are no longer re-exporting these?

Reply:
They're included above in `pub use wgt::*;`